src/lightning/fabric/wrappers.py

```diff
@@ -48,6 +48,8 @@ def __init__(self, optimizer: Optimizer, strategy: Strategy, callbacks: Optional
             optimizer: The optimizer to wrap
             strategy: Reference to the strategy for handling the optimizer step
 
+        :meta public:
```

Comment:
What does this do? It seems unnecessary. Let's keep proteced APIs hidden, and docs are just there for the developers.

Reply:
yeah, remove my bad...
```suggestion
```